nixos/tests/perses.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, ... }:
```

Comment:
Would be nice if this test could be registered in the package's `passthru.tests`